src/index.html

```diff
@@ -1,15 +1,101 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>HTML Form</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>HTML Form</h1>
-  <script type="text/javascript" src="./main.js"></script>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>HTML Form</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <form action="" method="post">
+      <!-- Personal Information -->
+      <fieldset class="block">
+        <legend>Personal information</legend>
+        <label class="block-item">
+          Surname:
+          <input type="text" name="surname" autocomplete="off">
+        </label>
+        <label class="block-item">
+          Name:
+          <input type="text" name="name" autocomplete="off">
+        </label>
+        <label class="block-item">
+          How old are You?
+          <input type="number" class="age-field" name="age" value="12" min="12" max="100">
```

Comment:
в тебе немає вже цього класу
```suggestion
          <input type="number" name="age" value="12" min="12" max="100">
```